modules/oadp-installing-dpa-1-2-and-earlier.adoc

```diff
@@ -342,6 +342,8 @@ spec:
 endif::[]
 
 . Click *Create*.
+
+.Verification
```

Comment:
```suggestion
[id="verifying-oadp-installation_{context}"]
== Verifying the installation
```

re: Wanting to link to this part of the doc: how about we change this to a header and use that as the anchor for the xref link in the Upgrade notes for 1.3 & 1.2

PS. the title is a bit crap, but you can come up with something much better

Reply:
@anarnold97 
That's a great idea. I hope that OCP goes along with it.